### PR TITLE
fix

### PR DESCRIPTION
**Pull request by mrbuds**

### CODE REVIEW

File: WeakAuras/GenericTrigger.lua [hunk 1]
```diff
@@ -1131,8 +1131,8 @@ function HandleEvent(frame, event, arg1, arg2, ...)
   end
   if (event == "PLAYER_ENTERING_WORLD") then
     timer:ScheduleTimer(function()
-      Private.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
       HandleEvent(frame, "WA_DELAYED_PLAYER_ENTERING_WORLD", arg1, arg2)
+      Private.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
```
Comment (mrbuds) [inline WeakAuras/GenericTrigger.lua:1135]: Private.StartProfileSystem on this event is already done in the HandleEvent call

Comment (InfusOnWoW) [inline WeakAuras/GenericTrigger.lua:1135]: this seems unintentional

Comment (mrbuds) [reply WeakAuras/GenericTrigger.lua:1135]: HandleEvent run StartProfileSystem / StopProfileSystem on "generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD" itself, so figured there is no need to do it on there

Comment (mrbuds) [reply WeakAuras/GenericTrigger.lua:1135]: (yes it's slightly offtopic for the description of this commit)

**Review by InfusOnWoW — APPROVED** [review]

**Pull request merged**